layouts/default.vue

```diff
@@ -5,49 +5,4 @@
 </template>
 
 <style lang="scss">
-html
```

Comment:
Maybe we should drop the `oku-circ.vue` layout and just move those styles here to the default layout? Maybe you plan on doing that next as part of styling the oku circ screen?